main.go

```diff
@@ -1039,6 +1039,12 @@ func parseFlags() (config, error) {
 			return cfg, fmt.Errorf("only one of --traces.read.endpoint and --experimental.traces.read.endpoint-template allowed")
 		}
 
+		if !strings.Contains(cfg.traces.readTemplateEndpoint, "{tenant}") {
+			fmt.Fprintf(os.Stderr,
```

Comment:
I would change this to `level.Warn(logger).Log(...)`

Reply:
The problem here is that the problem is detected in parseFlags(), but until parseFlags() is completed we can't create the logger because don't know the level.

I can probably move the check outside of parseFlags completely if you think that will help.

Reply:
ah you are right. I think then makes sense to leave inside `parseFlags(..)` :+1: